testsuites/syncgateway/functional/tests/conftest.py

```diff
@@ -168,6 +168,24 @@ def params_from_base_suite_setup(request):
         log_info("Using document storage for sync meta data")
         persist_cluster_config_environment_prop(cluster_config, 'xattrs_enabled', False)
 
+    try:
+        server_version
+    except NameError:
+        log_info("Server version is not provided")
+        persist_cluster_config_environment_prop(cluster_config, 'server_version', "")
+    else:
+        log_info("Running test with server version {}".format(server_version))
+        persist_cluster_config_environment_prop(cluster_config, 'server_version', server_version)
+
+    try:
+        sync_gateway_version
```

Comment:
Not sure if this is needed. If sync_gateway_version is not specified, it will blow up at line 117: check_xattr_support(server_version, sync_gateway_version). We always specify sync_gateway_version. I don't understand the motivation behind adding sync_gateway_version: "", to the cluster config.